aten/src/ATen/native/TensorCompare.cpp

```diff
@@ -451,15 +486,13 @@ std::tuple<Tensor, Tensor> max(const Tensor& self, int64_t dim, bool keepdim) {
 }
 
 std::tuple<Tensor, Tensor> min(const Tensor& self, int64_t dim, bool keepdim) {
+  // CUDA and CPU dispatch keys are handled by the structured kernel implementation.
+  TORCH_INTERNAL_ASSERT(self.is_quantized());
```

Comment:
Same as last PR, I think it's fine to remove this (and rename `min()` to imply that it's a quantized kernel)